conf/config-default.yaml

```
@@ -69,6 +69,11 @@ apisix:
         memory_size: 50m
 
   delete_uri_tail_slash: false    # delete the '/' at the end of the URI
+
+
+  #http is the default proxy mode. proxy_mode can be oneOf http or stream or http&stream
```

```suggestion
  #http is the default proxy mode. proxy_mode can be one of `http` or `stream` or `http&stream`
```
It confuses me how to configure `http&stream` without reading the source code. Reading the description in config-default.yaml alone is not enough.

---

Please add the test case.